docs/reference/components/sinks.cue

```diff
@@ -204,9 +204,14 @@ components: sinks: [Name=string]: {
 						examples: []
 						options: {
 							in_flight_limit: {
```

Comment:
Don't we also need to add the auto-concurrency-specific options here as well?

Reply:
Of course, will do.